pom.xml

```diff
@@ -1072,18 +1072,6 @@
                                 </configuration>
                             </execution>
 
-                            <execution>
-                                <phase>initialize</phase>
-                                <id>gulp clean</id>
-                                <goals>
-                                    <goal>npm</goal>
-                                </goals>
-                                <configuration>
-                                    <!-- The package.json must define an "mvnclean" script -->
-                                    <arguments>run mvnclean</arguments>
```

Comment:
Hmm, intent of this was to remove jar file during gulp clean. Which sounds ok considering generated jar file includes result of gulp build or maybe I misunderstand this process?

Reply:
ehh. not sure why this was here, I didn't add it, did I? :)

Reply:
The intent wasn't to remove the .jar file, I was trying to clear out the stuff in `target` that was copied by maven-resources-plugin. Deleting all of `target` was clearly a bad idea, I should have individually cleaned out the folders copied it, just more of a maintenance headache.